src/rebar3_typer_prv.erl

```diff
@@ -10,11 +10,53 @@
               {rebar_state, command_parsed_args, 1}]).
 
 -define(PROVIDER, typer).
--define(OPTS,
-        [{boolean, $b, "boolean", {boolean, false}, "Boolean example"},
-         {string, $s, "rebar-config", {string, "some.txt"}, "String example"},
-         {atom, $a, "ignore", atom, "Atom example"},
-         {atom_none, $n, "atom-none", {atom, none}, "Atom example, with default"}]).
+-define(OPTS, %% TODO: consider adding shorthand version to some (or all) options, even if it doesn't exist on TypEr itself
+        %% list of {Name,  ShortOpt,  LongOpt,    ArgSpec,   HelpMsg}
+        [{recursive,
+          undefined,
+          "recursive",
+          {boolean, false},
+          "Search directories recursively for .erl files below them."},
+         {show,
+          undefined,
+          "show",
+          {boolean, false},
+          "Print type specifications for all functions on stdout. (This is the default behaviour; this option is not really needed.)"},
+         {show_exported,
+          undefined,
+          "show-exported",
+          {boolean, false},
+          "Same as --show, but print specifications for exported functions only. Specs are displayed sorted alphabetically on the function's name."},
+         {annotate,
+          undefined,
+          "annotate",
+          {boolean, false},
+          "Annotate the specified files with type specifications."},
+         {annotate_inc_files,
+          undefined,
+          "annotate-inc-files",
+          {boolean, false},
+          "Same as --annotate but annotates all -include() files as well as all .erl files. (Use this option with caution - it has not been tested much)."},
+         {edoc,
+          undefined,
+          "edoc",
+          {boolean, false},
+          "Print type information as Edoc @spec comments, not as type specs."},
+         {plt,
+          undefined,
+          "plt",
+          string,
+          "Use the specified dialyzer PLT file rather than the default one."},
+         {typespec_files,
+          $T,
+          undefined,
+          string,
+          "The specified file(s) already contain type specifications and these are to be trusted in order to print specs for the rest of the files. (Multiple files or dirs, separated by spaces, can be specified.)"},
```

Comment:
Does this actually work for rebar3? I mean… _multiple files separated by spaces_ … I don't think that's how rebar3 command line parsing thing works.

Reply:
I copy and pasted the whole descriptions with no regards to fixing them yet. rebar3 can handle multiple files, as long as they are separated by commas and not spaces, so this is just yet another option that has to be updated :)

Reply:
So, let's go ahead and fix them :)